pom.xml

```diff
@@ -102,5 +108,16 @@
             <version>1.50</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>credentials</artifactId>
+            <version>2.2.1-rc687.f3efa56bfbe1</version>
```

Comment:
TODO: This will likely be released in 2.3.0; 2.2.1 will end up being a minor release before that.